--- wget-1.8.1/src/utils.c.orig	Mon Dec 17 17:05:08 2001
+++ wget-1.8.1/src/utils.c	Tue Feb  5 18:16:18 2002
@@ -452,9 +452,11 @@
   else if (pid != 0)
     {
       /* parent, no error */
+      if (! opt.quiet) {
       printf (_("Continuing in background, pid %d.\n"), (int)pid);
       if (changedp)
 	printf (_("Output will be written to `%s'.\n"), opt.lfilename);
+      }
       exit (0);			/* #### should we use _exit()? */
     }
 
